plank/laravel-media-manager
# Prevent reload when detaching media

### Overview
1. When removing an attached media the MM forces a page reload causing un saved values from an article to be discarded
2. When creating media carousel using MM we weren't attaching some accessibility attributes

### Work was done
1.  when removing an attachment, we leverage the custom event we created for attaching media to not trigger a reload and update the media previews 
2. created a helper function that'll return attribute values when called, we use it in both the copy carousel function and copy html function in the side-panel for individual media 


### To test

1. navigate to an[ article with attached media ](https://tce.7.plankdesign.com/admin/articles/86149/edit)
2. open the media manager using the the media preview
3. a side panel should open along with media manger where you can remove the attachment 
4. remove attachment, sidepanel should close, a success message will appear
5. close media manager, and the attached media should be gone :) 

closes #75 
closes #76
